repo/meta/schema/package-schema.json

```diff
@@ -114,7 +114,8 @@
           "type": "string"
         },
         "version": {
-          "type": "string"
+          "type": "string",
+          "pattern": "^[-a-zA-Z0-9.]+$"
```

Comment:
Did you check that this is compatible with all versions currently in Universe? Or is this just the characters allowed by semver?